### ISSUE TITLE
Update experiment page warning for `locales` mechanism

### ISSUE THREAD

**Issue by ckprice** [mozilla/testpilot#1959]
STR:

- Add a `locales` field to experiment page YAML for `de` ([ref](https://github.com/mozilla/testpilot/blob/master/docs/content/reference.md#locales)).

```
locales:
  - 'de'
```

- Set language to `en`
- Visit experiment page

Expected Result:

- Message states that the experiment is *not* available in English.

OR

- Message states that the experiment is *only* available in German.

Actual Result:

- Message states that the experiment is *only* available in English.

Looks like this string has been hard-coded to English regardless of what's in the `locales` field.

**Comment by johngruen** [comment]
@ckprice yeah, that's what happened...

should probably go with the negative case: 

"this experiment is not available in [FIRST CHOICE LANGUAGE]"

**Comment by ckprice** [comment]
So, I think we'd need to update https://github.com/mozilla/testpilot/blob/master/locales/en-US/app.ftl#L222 from

> This experiment is only available in English.

to something like

> This experiment is not available in `{$lang}`

(don't know if you can reference a user's language like that in the app)

**Comment by jaredhirsch** [comment]
@ckprice Actually, since you're localizing the error message, you can just use a placeholder to ask localizers to put their language's name in that spot:

```
# LOCALIZATION NOTE: Replace {{lang}} with the localized name of the user's language.
localeUnavailableWarningTitle = This experiment is not available in {{lang}}.
```

I'm not doing a great job of wording the localization note, but that's the right format for the FTL file. See the wiki for more: https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Add_localization_notes

**Comment by jaredhirsch** [comment]
Note that I changed the name of the FTL key, because I think we need to change the keys to retranslate a string. So, there's probably a tiny bit of work to change the l10n-id in the template, too.

**Comment by ckprice** [comment]
@6a68 thanks! Opened a PR with some of your comments included in #1969
